cpp/roaring.hh

```diff
@@ -39,7 +40,10 @@ A C++ header for Roaring Bitmaps.
 
 namespace roaring {
 
-class RoaringSetBitForwardIterator;
+class RoaringSetBitBiDirectionalIterator;
+
+/** DEPRECATED, use `RoaringSetBitBiDirectionalIterator`. */
+using RoaringSetBitForwardIterator = RoaringSetBitBiDirectionalIterator;
```

Comment:
Can we use `CROARING_DEPRECATED`?

Reply:
I'd like to, but:
1. currently, the `Roaring64MapSetBitForwardIterator` is not marked as deprecated.
2. `CROARING_DEPRECATED` expands to `__attribute__((deprecated))` for GNUC and Clang, so `using xxx CROARING_DEPRECATED = xxx;` works. However, for MSVC, it expands to `__declspec(deprecated)`, which won't work in this format.